src/pages/IssueCollection/components/IssueCollectionForm.tsx

```diff
@@ -0,0 +1,156 @@
+import classNames from 'classnames';
+import { Button, MxLink } from 'components';
+import { CollectionTypeByNftEnum, DataTestIdsEnum } from 'localConstants';
+import { routeNames } from 'routes';
+import { NftEnumType } from 'types';
+import { useIssueCollectionForm } from '../hooks';
+import { IssueCollectionFieldsEnum } from '../types';
+
+export const IssueCollectionForm = () => {
+  const formik = useIssueCollectionForm();
```

Comment:
why not destructure?

Reply:
There is nothing to destructure. It is the formik object